cpp/ql/src/semmle/code/cpp/Class.qll

```diff
@@ -8,7 +8,22 @@ private import semmle.code.cpp.internal.ResolveClass
  * A class type [N4140 9].
  *
  * While this does include types declared with the `class` keyword, it also
- * includes types declared with the `struct` and `union` keywords.
+ * includes types declared with the `struct` and `union` keywords.  For example:
```

Comment:
I think it is worth being more explicit:
```
For example, the types `MyClass`, `MyStruct` and `MyUnion` in
```

Reply:
Done.